code/datums/bark.dm

```diff
@@ -96,6 +96,95 @@
 	id = "coggers"
 	soundpath = 'sound/machines/clockcult/integration_cog_install.ogg' //Yet another unexpectedly good bark sound
 
+/datum/bark/banjoahuh
+	name = "Ahuh"
+	id = "banjokazooie"
+	soundpath = 'sound/voice/barks/ahuh.ogg'
+
+/datum/bark/bleat_bark
+	name = "Bleat"
+	id = "Bleat"
+	soundpath = 'sound/voice/barks/bleat_bark.ogg'
+
+/datum/bark/bottalk1
+	name = "bot1"
+	id = "bottalk_1"
+	soundpath = 'sound/voice/barks/bottalk_1.ogg'
+
+/datum/bark/bottalk2
+	name = "bot2"
+	id = "bottalk_2"
+	soundpath = 'sound/voice/barks/bottalk_2.ogg'
+
+/datum/bark/bottalk3
+	name = "bot3"
+	id = "bottalk_3"
+	soundpath = 'sound/voice/barks/bottalk_3.ogg'
+
+/datum/bark/bottalk4
+	name = "bot4"
+	id = "bottalk_4"
+	soundpath = 'sound/voice/barks/bottalk_4.ogg'
+
+/datum/bark/bug
+	name = "bug"
+	id = "bug"
+	soundpath = 'sound/voice/barks/bug.ogg'
+
+/datum/bark/caw
+	name = "caw"
+	id = "caw"
+	soundpath = 'sound/voice/barks/caw.ogg'
+
+/datum/bark/caw2
+	name = "caw2"
+	id = "caw2"
+	soundpath = 'sound/voice/barks/caw2.ogg'
+
+/datum/bark/caw3
+	name = "caw3"
+	id = "caw3"
+	soundpath = 'sound/voice/barks/caw3.ogg'
+
+/datum/bark/mothsqueak
+	name = "mothsqueak"
+	id = "mothsqueak"
+	soundpath = 'sound/voice/barks/mothsqueak.ogg'
+
+/datum/bark/pugdog
+	name = "pug"
+	id = "pug"
+	soundpath = 'sound/voice/barks/pug.ogg'
+
+/datum/bark/pugdog2
+	name = "pug2"
+	id = "pug2"
+	soundpath = 'sound/voice/barks/pug2.ogg'
+
+/datum/bark/speak1
+	name = "speak_1"
+	id = "speak_1"
+	soundpath = 'sound/voice/barks/speak_1.ogg'
+
+/datum/bark/speak2
+	name = "speak_2"
+	id = "speak_2"
+	soundpath = 'sound/voice/barks/speak_2.ogg'
+
+/datum/bark/speak3
+	name = "speak_3"
+	id = "speak_3"
+	soundpath = 'sound/voice/barks/speak_3.ogg'
+
+/datum/bark/speak4
+	name = "speak_4"
+	id = "speak_4"
+	soundpath = 'sound/voice/barks/speak_4.ogg'
+
+/datum/bark/zap
+	name = "zap"
+	id = "zap"
+	soundpath = 'sound/voice/barks/zap.ogg'
```

Comment:
I'm unsure how this server handles modularization, but you can possibly move these to (for example), lumos/code/datums/bark.dm to separate the changes out for ease of upstream pulls.

The sounds themselves can also go into lumos/sound/voice/barks/???.ogg